changelog.md

```diff
@@ -39,6 +39,60 @@ would assign these ports as host ports if available in the
 processed offer. That misled people into thinking that these ports corresponded to host ports. The new code always
 randomizes host ports assignment without `"requirePorts"` or explicit `"hostPort"` configuration.
 
+##### Additional task statistics in the `/v2/apps` and `/v2/apps/{app id}` endpoints (Optional)
+
+If you pass `embed=apps.taskStats`/`embed=app.taskStats` as a query parameter, you get additional taskStats
+embedded in you app JSON.
+ 
+Task statistics are provided for the following groups of tasks. If no tasks for the group exist, no statistics are
+offered:
+
+* `"withLatestConfig"` contains statistics about all tasks that run with the same config as the latest app version.
+* `"startedAfterLastScaling"` contains statistics about all tasks that were started after the last scaling or
+  restart operation.
+* `"withOutdatedConfig"` contains statistics about all tasks that were started before the last config change which
+  was not simply a restart or scaling operation.
+* `"totalSummary"` contains statistics about all tasks.
+
+Example JSON:
+
+```javascript
+{
+  // ...
+  "taskStats": {
+    {
+      "startedAfterLastScaling" : {
+        "stats" : {
+          "counts" : { // equivalent to tasksStaged, tasksRunning, tasksHealthy, tasksUnhealthy
+            "staged" : 1,
+            "running" : 100,
+            "healthy" : 90,
+            "unhealthy" : 4
+          },
+          "lifeTime" : { // measured from the "stagedAt" timestamp of Mesos
+            "averageSeconds" : 20.0,
+            "medianSeconds" : 10.0
+          }
+        }
+      },
+      "withLatestConfig" : {
+        "stats" : { /* ... same structure as above ... */ }
+      },
+      "withOutdatedConfig" : {
+        "stats" : { /* ... same structure as above ... */ }
+      },
+      "totalSummary" : {
```

Comment:
If it is not really important to you, we will not change that anymore. Since the UI code is already released.